README.md

```diff
@@ -193,6 +193,45 @@ Git Operator pushes SDP entities on Kubernetes to a Git repository and create pu
 For the list of available Helm parameters, please go to [Helm Values](doc/values.md)
 
 # Advanced Usage
+## Configuring what entities to sync
+The SDP Operator supports different of ways of specifying what to sync and how:
+
+ 1. Configuring what tags to sync
+ 2. Configuring what entity types to sync
+
+Note that both can be used together.
+
+### Configuring what tags to sync
+We can specify what tags the operator will work on via the environemnt variable `APPGATE_OPERATOR_TARGET_TAGS`. This variable supports a list of tags (separated by comma) to sync. This means that entities that have any of those tags will be managed by the operator.
+
+Example:
+
+``` shell
+APPGATE_OPERATOR_TARGET_TAGS="devops,dev"
+```
+
+With that environemnt variable defined the operator will only modify or delete entities that any of the tags devops or dev (ot both at the same time).
+
+We can also specify explicitly that we don't want to manage entities with specific tags. This can be done with the environemnt variable `APPGATE_OPERATOR_EXCLUDE_TAGS`. Entities with any of the tags listed there won't be modified or deleted.
```

Comment:
Can you add a sentence to clarify what takes precedence between target tags and exclude tags.
What happens if an entity is tagged with both a target tag and an excluded tag, will it be excluded?